tools/askor/askor_de.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="askor_de" name="AskoR_DE" version="0.01">
+    <description>Edger for askomics </description>
+    <requirements>
+        <requirement type="package" version="3.34.6">bioconductor-limma</requirement>
+        <requirement type="package" version="1.4.29">r-statmod</requirement>
+        <requirement type="package" version="3.20.7">bioconductor-edger</requirement>
+        <requirement type="package" version="2.2.1">r-ggplot2</requirement>
+        <requirement type="package" version="1.1_2">r-rcolorbrewer</requirement>
+        <requirement type="package" version="0.7.0">r-ggrepel</requirement>
+        <requirement type="package" version="3.0.1">r-gplots</requirement>
+        <requirement type="package" version="1.3.2">r-optparse</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+      export ASKOR_PATH='$__tool_directory__' &&
+      Rscript '$__tool_directory__/Asko_DE.R'
+      --out ${out_pdf}
+      --outdir askor_output
+      --fileofcount ${fileofcounts}
+      --samples ${samples}
+      --contrasts ${contrasts}
+      --org ${organism}
+      --col_genes ${colgenes}
```

Comment:
Il you use section in your inputs, you must to change your variable.

https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-inputs-section